packages/styled-components/src/models/StyledNativeComponent.js

```diff
@@ -244,3 +251,10 @@ export default (InlineStyle: Function) => {
 
   return createStyledNativeComponent;
 };
+
+function isArrayEqual(a: any, b: any) {
```

Comment:
@zhigang1992 Any specific reason for not adding the array type in the function argument itself? 🤔

Reply:
Because `previousStyle` is typed `any?`